src/components/AboutPage/index.tsx

```diff
@@ -0,0 +1,100 @@
+                        results converted into the Simularium file format
+                        (JSON). The Simularium Viewer uses advanced rendering
+                        techniques developed by Le Muzic et al. (2015) and
+                        Mindek et al. (2017) to enable meaningful interpretation
+                        of spatial relationships among thousands of moving
+                        components. Users can import a trajectory file (JSON
+                        format) from their computer, or they can stream provided
+                        examples by choosing one from the Load model dropdown or
+                        by clicking on a card above. Future versions of
+                        Simularium will allow you to modify, run, and even
+                        create simulations.
+                    </p>
+                    <h3>Current object representtion options:</h3>
+                    <p>
+                        Spheres: by default, each agent in a scene is
+                        represented as a single sphere Mesh surfaces: represent
+                        each agent as a mesh file, e.g. coarse molecular
+                        surfaces Multi-sphere: provide Protein Databank .pdb
+                        files Line representations for fibers, filaments, or
+                        bonds
+                    </p>
+                    <h3>Upcoming plans</h3>
+                    <p>
+                        Volume rendering for RDME or PDE-based simulation
+                        results Support for .cif files and coarse-grain
+                        sphereTree files for multi-sphere rendering
+                    </p>
+                </div>
+                <div className={styles.imageContainer}>
+                    <img src={flowchartImg} alt="Flowchart of Simularium" />
+                    <img
+                        src={agentRepOptionsImg}
+                        alt="Agent representation options"
+                    />
+                </div>
+            </ContentPagePanel>
+
+            <ContentPagePanel isDark={true}>
+                <h2>Future plans for Simularium</h2>
+                <h3>
+                    Create. modify, run, experiment, share, interoperate, & grow
+                    as a community
+                </h3>
+                <p>
+                    In the next phase of its development, Simularium will enable
+                    users to modify simulation parameters for provided models
+                    through the web-based interface, to run the modified
+                    simulations on the cloud, and to analyze the results.
+                    Simularium currently supports CytoSim and ReaDDy. We plan to
+                    wrap several published packages that can serve as templates
+                    for community development and to encourage growth of the
+                    system by providing a well- documented API for simulation
```

Comment:
```suggestion
                    system by providing a well-documented API for simulation
```
You have the copy from the Figma design, but it looks weird to me.